watertap/core/zero_order_costing.py

```diff
@@ -856,28 +857,9 @@ def cost_landfill(blk):
         # Determine if a costing factor is required
         factor = parameter_dict["capital_cost"]["cost_factor"]
 
-        # Add cost variable and constraint
-        blk.capital_cost = pyo.Var(
-            initialize=1,
-            units=blk.config.flowsheet_costing_block.base_currency,
-            bounds=(0, None),
-            doc="Capital cost of unit operation")
-
-        expr = pyo.units.convert(
-            (A*(blk.unit_model.total_mass[t0] / blk.unit_model.capacity_basis[t0])**B),
-            to_units=blk.config.flowsheet_costing_block.base_currency)
-
-        if factor == "TPEC":
-            expr *= blk.config.flowsheet_costing_block.TPEC
-        elif factor == "TIC":
-            expr *= blk.config.flowsheet_costing_block.TIC
-
-        blk.capital_cost_constraint = pyo.Constraint(
-            expr=blk.capital_cost == expr)
-
-        # Register flows
-        blk.config.flowsheet_costing_block.cost_flow(
```

Comment:
Electricity demand still needs to be registered in the main method - the general power law method only does the capital cost part.

Reply:
Ohh, I registered the electricity demand and corresponding tests accordingly.